### PR TITLE
Added backing address to tokens `USDC USDT BTC BCH ETH DOGE LTC`

### PR DESCRIPTION
**Pull request by siradji**
#### What kind of PR is this?:

/kind feature

#### Which issue(s) does this PR fixes?:

Fixes #

#### Sample Links & Screenshots:

Link: 
<details>
<summary>Desktop Screenshot</summary>

</details>
<details>
<summary>Mobile Screenshot</summary>

</details>

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Tested on multiple web browsers
- [ ] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [ ] No console errors
- [ ] Unit tests*
- [ ] Added e2e tests*

### CODE REVIEW

File: src/pages/tokens/[id].tsx [hunk 1]
```diff
@@ -141,6 +156,30 @@ function ListLeft ({ token }: { token: TokenData }): JSX.Element {
   )
 }
 
+function BackingAddress ({ tokenName }: {tokenName: string}): JSX.Element {
+  const isEth = tokenName === TokensWithBackingAddress.ETH || tokenName === TokensWithBackingAddress.USDC || tokenName === TokensWithBackingAddress.USDT
```
Comment (joeldavidw) [inline src/pages/tokens/[id].tsx:160]: You could use _fall-through_ in the `switch` `case` to standardise without the need of the if check.

File: src/pages/tokens/[id].tsx [hunk 2]
```diff
@@ -141,6 +156,30 @@ function ListLeft ({ token }: { token: TokenData }): JSX.Element {
...
+        switch (tokenName) {
+          case TokensWithBackingAddress.BCH:
+            return <AddressLinkExternal url='https://www.blockchain.com/bch/address/38wFczGqaaGLRub2U7CWeWkMuPDwhMVMRf' text='38wFczGqaaGLRub2U7CWeWkMuPDwhMVMRf' testId='BackingAddress.BCH' />
+          case TokensWithBackingAddress.LCH:
+            return <AddressLinkExternal url='https://live.blockcypher.com/ltc/address/MLYQxJfnUfVqRwfYXjDJfmLbyA77hqzSXE' text='MLYQxJfnUfVqRwfYXjDJfmLbyA77hqzSXE' testId='BackingAddress.LCH' />
+          case TokensWithBackingAddress.DOGE:
+            return <AddressLinkExternal url='https://dogechain.info/address/D7jrXDgPYck8jL9eYvRrc7Ze8n2e2Loyba' text='D7jrXDgPYck8jL9eYvRrc7Ze8n2e2Loyba' testId='BackingAddress.DOGE' />
+          case TokensWithBackingAddress.BTC:
+            return <AddressLinkExternal url='https://www.blockchain.com/btc/address/38pZuWUti3vSQuvuFYs8Lwbyje8cmaGhrT' text='38pZuWUti3vSQuvuFYs8Lwbyje8cmaGhrT' testId='BackingAddress.BTC' />
+        }
```
Comment (joeldavidw) [inline src/pages/tokens/[id].tsx:177]:
```suggestion
        case TokensWithBackingAddress.ETH:
        case TokensWithBackingAddress.USDC:
        case TokensWithBackingAddress.USDT:
            return <AddressLinkExternal url='https://etherscan.io/address/0x94fa70d079d76279e1815ce403e9b985bccc82ac' text='0x94fa70d079d76279e1815ce403e9b985bccc82ac' testId='BackingAddress.ETH' />
        }
```

**Review by joeldavidw — CHANGES_REQUESTED** [review]

File: src/pages/tokens/[id].tsx [hunk 3]
```diff
@@ -6,14 +6,24 @@ import { TokenData } from '@defichain/whale-api-client/dist/api/tokens'
...
-import { AddressLink } from '@components/commons/AddressLink'
+import { AddressLink, AddressLinkExternal } from '@components/commons/AddressLink'
 import { TxIdLink } from '@components/commons/TxIdLink'
 import { isNumeric } from '../../utils/commons/StringValidator'
 
 interface TokenAssetPageProps {
   token: TokenData
 }
 
+enum TokensWithBackingAddress {
```
Comment (joeldavidw) [inline src/pages/tokens/[id].tsx:17]: With the switch statement this can be removed.

File: src/pages/tokens/[id].tsx [hunk 4]
```diff
@@ -141,6 +156,34 @@ function ListLeft ({ token }: { token: TokenData }): JSX.Element {
...
+  return (
+    <AdaptiveList.Row name='Backing Address' className='break-all'>
+      {(() => {
+        if (isEth) {
+          return <AddressLinkExternal url='https://etherscan.io/address/0x94fa70d079d76279e1815ce403e9b985bccc82ac' text='0x94fa70d079d76279e1815ce403e9b985bccc82ac' testId='BackingAddress.ETH' />
+        }
+        switch (tokenName) {
+          case TokensWithBackingAddress.BCH:
+            return <AddressLinkExternal url='https://www.blockchain.com/bch/address/38wFczGqaaGLRub2U7CWeWkMuPDwhMVMRf' text='38wFczGqaaGLRub2U7CWeWkMuPDwhMVMRf' testId='BackingAddress.BCH' />
+          case TokensWithBackingAddress.LCH:
```
Comment (joeldavidw) [inline src/pages/tokens/[id].tsx:171]: LTC not LCH

**Review by joeldavidw — CHANGES_REQUESTED** [review]

File: src/pages/tokens/[id].tsx (hunk 3 above)
Comment (siradji) [reply src/pages/tokens/[id].tsx:17]: well, it is also used here https://github.com/DeFiCh/scan/blob/a7535e69e17cb554ce3861ae9fc90efee36838e7/src/pages/tokens/%5Bid%5D.tsx#L112-L116

**Pull request merged**